pkg/depository/create.go

```diff
@@ -17,18 +17,176 @@ limitations under the License.
 package depository
 
 import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
 	"fmt"
+	"github.com/bestchains/bestchains-contracts/library/context"
+	"io"
+	"k8s.io/klog/v2"
+	"net/http"
+	"strconv"
+	"time"
 
+	"github.com/bestchains/bc-cli/pkg/common"
+	"github.com/bestchains/bestchains-contracts/library"
 	"github.com/spf13/cobra"
 )
 
 func NewCreateDepositoryCmd() *cobra.Command {
+	var (
+		name        string
+		contentType string
+		contentID   string
+		platform    string
+	)
+
 	cmd := &cobra.Command{
 		Use: "depository NAME [args]",
-		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("create depository")
+		RunE: func(cmd *cobra.Command, args []string) error {
+			klog.V(5).Infof("create depository")
+
+			n, err := cmd.Flags().GetString("name")
+			if err != nil {
+				return err
+			}
+			t, err := cmd.Flags().GetString("contentType")
+			if err != nil {
+				return err
+			}
+			id, err := cmd.Flags().GetString("contentID")
+			if err != nil {
+				return err
+			}
+			p, err := cmd.Flags().GetString("platform")
+			if err != nil {
+				return err
+			}
+
+			// Generate ValueDepository
+			valDep := common.ValueDepository{
+				Name:             n,
+				ContentType:      t,
+				ContentID:        id,
+				TrustedTimestamp: strconv.FormatInt(time.Now().Unix(), 10),
+				Platform:         p,
+			}
+			// Marshal & encoding
+			rawVal, err := json.Marshal(valDep)
+			if err != nil {
+				return err
+			}
+
+			value := base64.StdEncoding.EncodeToString(rawVal)
+
+			// Obtain private & public key; generating key
+			// TODO: key auth
+			privateKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
1. we should save private key and public key to local file `~/.bestchains/{account_address}.json

2. add arguments for this command to use this key pair 
```
./bc-cli create depository --wallet ~/.bestchains --wallet-account 0xsklj34234fsf --wallet-account-generate
```
-  if account provided,we use `PutValue`  
- if --wallet-account-generate is specified , auto-generate account and use this account in `PutValue`
- if account not provided,we use `PutUntrustValue`